vignettes/babelquarto.html

```diff
@@ -0,0 +1,710 @@
+<!DOCTYPE html>
```

Comment:
Could you please remove the preview from the PR?

Reply:
Hopefully I did that properly!